tests/testthat/test-apply_row_grp_plan.R

```diff
@@ -0,0 +1,160 @@
+library(dplyr)
+library(tidyr)
```

Comment:
This should be in the tests/testthat.R script IMO. Loading libraries within a particular test script can make the testing environment dirty. Then we all have access to dplyr/tidyr